### PR TITLE
Update BBEs for AccessLog Improvements

### PR DESCRIPTION
**Pull request by AzeemMuzammil**
Tracked Issue: https://github.com/ballerina-platform/ballerina-library/issues/6726

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: examples/http-access-logs/http_access_logs.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # HTTP service - Access logs
 
-Ballerina allows enabling HTTP access logs, which can be used to record the HTTP requests handled by the application. HTTP access logs are disabled by default. Set `console=true` under `ballerina.http.accessLogConfig` in the `Config.toml` file to enable them. Additionally, the `path` field can be used to specify the file path to save the access logs.
+Ballerina allows enabling HTTP access logs, which can be used to record the HTTP requests handled by the application. HTTP access logs are disabled by default. To enable them, set `console=true` under `ballerina.http.accessLogConfig` in the `Config.toml` file. The optional `format` field allows you to specify the format of the access logs, with options being `flat` or `json`, defaulting to `flat`. You can also optionally specify which attributes to log by providing a list in the `attributes` field. Additionally, the optional `path` field can be used to specify the file path to save the access logs.
```
Comment (TharmiganK) [inline examples/http-access-logs/http_access_logs.md:3]:
```suggestion
Ballerina allows enabling HTTP access logs, which can be used to record the HTTP requests handled by the application. HTTP access logs are disabled by default. To enable them, set `console=true` under `ballerina.http.accessLogConfig` in the `Config.toml` file. Additionally, the path field can be used to define the file path for saving the access logs, offering flexible log management. The log format can be specified using the optional `format` field, with `flat` and `json` as available options, defaulting to `flat`. Furthermore, you can customize the logged attributes by listing them in the optional `attributes` field.
```

File: examples/http-access-logs/Config.toml [hunk 2]
```diff
@@ -1,5 +1,9 @@
 [ballerina.http.accessLogConfig]
 # Enable printing access logs on the Console. The default value is `false`.
 console = true
+# Specify the format of access logs. Options are `flat` or `json`. The default value is `flat`.
+format = "flat"
+# List of attributes to include in the access logs. This field is optional.
+attributes = ["ip", "date_time", "request", "status", "response_body_size", "http_referrer", "http_user_agent"]
```
Comment (dilanSachi) [inline examples/http-access-logs/Config.toml:7]: Shall we describe what the behavior is when we add the `attributes=[]` or no attributes field. Wdyt?

Comment (TharmiganK) [reply examples/http-access-logs/Config.toml:7]: @AzeemMuzammil  It is given in the spec right?

Comment (AzeemMuzammil) [reply examples/http-access-logs/Config.toml:7]: Yes, the values that are supported part of the attributes are specified in the spec

**Review by TharmiganK — COMMENTED** [review]
LGTM

File: examples/http-access-logs/http_access_logs.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # HTTP service - Access logs
 
-Ballerina allows enabling HTTP access logs, which can be used to record the HTTP requests handled by the application. HTTP access logs are disabled by default. Set `console=true` under `ballerina.http.accessLogConfig` in the `Config.toml` file to enable them. Additionally, the `path` field can be used to specify the file path to save the access logs.
+Ballerina allows enabling HTTP access logs, which can be used to record the HTTP requests handled by the application. HTTP access logs are disabled by default. To enable them, set `console=true` under `ballerina.http.accessLogConfig` in the `Config.toml` file. Additionally, the path field can be used to define the file path for saving the access logs, offering flexible log management. The log format can be specified using the optional `format` field, with `flat` and `json` as available options, defaulting to `flat`. Furthermore, you can customize the logged attributes by listing them in the optional `attributes` field.
```
[the remark this answers is not in the document]
Comment (MaryamZi) [reply examples/http-access-logs/http_access_logs.md:3]: I would leave the previous sentence as is.

[the remark this answers is not in the document]
Comment (TharmiganK) [reply examples/http-access-logs/http_access_logs.md]: Fixed via [cab5906](https://github.com/ballerina-platform/ballerina-distribution/pull/5504/commits/cab590687da6adbbd2a6be2d6c211a32d74568fe)

**Pull request merged**